core/sail/elasticsearch-store/pom.xml

```diff
@@ -94,5 +94,34 @@
 			<version>${project.version}</version>
             <scope>test</scope>
         </dependency>
-    </dependencies>
+		<dependency>
```

Comment:
I think your xml indentation settings might be out of date (or perhaps it's the previous edit on this file that went wrong). See https://github.com/eclipse/rdf4j/blob/master/.github/CONTRIBUTING.md#xml-indentation . There's a maven command to fix it for you - though it might be a good idea to adjust your editor settings as well.